utils/xsc/analyticsmetrics_test.go

```diff
@@ -122,3 +105,33 @@ func TestAnalyticsMetricsService_createAuditResultsFromXscAnalyticsBasicGeneralE
 		})
 	}
 }
+
+func getDummyContentForGeneralEvent(withJas, withErr bool) *results.SecurityCommandResults {
```

Comment:
Upon use you provide numbers to findings to compare to, but is it unclear what this Dummy function suppose to generate according to its arguments.
Please add a description to explain what is the expected output when using this func

Reply:
added comment